JS/Lesson35/tugba/my-app/src/components/RecipeList/index.jsx

```diff
@@ -1,18 +1,25 @@
-import { Link } from 'react-router-dom';
+import React from 'react';
+import { List, ListItem, Typography } from '@mui/material';
+import { Link, useNavigate } from 'react-router-dom';
 
-export const RecipeList = ({ recipes }) => {
-	return (
-		<div>
-			<h2>Search Results</h2>
-			<ul>
-				{recipes.map((recipe) => {
-					return (
-						<li key={recipe.idMeal}>
-							<Link to={`recipes/${recipe.idMeal}`}>{recipe.strMeal}</Link>
-						</li>
-					);
-				})}
-			</ul>
-		</div>
-	);
+const RecipeList = ({ recipes }) => {
+  const navigate = useNavigate();
+
+
+  return (
+    <div>
+      <Typography variant="h5">Search Results</Typography>
+      <List>
```

Comment:
This page can also be improved using MUI. For examples, https://mui.com/material-ui/react-card/#primary-action component could be used to display each recipe data in the list instead of plain text.